keyboards/momokai/aurora/readme.md

```diff
@@ -1,12 +1,14 @@
 # Momokai Aurora
 
+![Momokai Aurora](https://i.imgur.com/A7iHqzA.jpg)
```

Comment:
```suggestion
![Momokai Aurora](https://i.imgur.com/A7iHqzAh.jpg)
```

Reply:
explain

Reply:
Please see `readme.me` section of [PR checklist](https://docs.qmk.fm/#/pr_checklist)

Reply:
got it, thanks!